app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx

```diff
@@ -58,11 +83,22 @@ export default class LoginLedgerNanoS extends React.Component<Props, State> {
     if (progress !== nextProps.progress || error !== nextProps.error) {
       this.setState(this.computeStateFromProps(nextProps))
     }
+
+    if (this.props !== LOADED && nextProps.progress === LOADED) {
+      this.fetchInitialKeys()
+    }
   }
 
   computeStateFromProps = (props: Props) => {
     if (props.progress === LOADED) {
-      return { ledgerStage: CONNECTED, isLoading: false }
+      if (this.intervalId) {
+        // $FlowFixMe
```

Comment:
why the flow fix?

Reply:
nit: no need to check that `intervalId` exists since `clearInterval` handles `undefined` value silently.